### DISCUSSION TITLE
Optional display style of circuit diagram

### DISCUSSION THREAD

**Discussion by GrimPixel** [IndiePython/nodezator#68] [Ideas]
Just follow the rules of drawing a circuit diagram. Nodes are like chips and links are like wires.

**Comment by KennedyRichard** [comment]
I assume you are referring to the graphical elements, not the structure of a circuit. I assume that because a circuit, by definition, cannot represent a direct acyclic graph (DAG), the mathematical concept upon which node layouts are baded. That is, node graphs are, at least usually, not cyclic, unlike circuits.

Sticking to this assumption, then yes, a simpler graphical representation that allowed us to quickly overview the structure of the node graph would indeed by very useful.

In fact, I believe we already have a planned feature that targets this need, which is the overview/map feature described in this comment: https://github.com/IndiePython/nodezator/discussions/14#discussioncomment-3057031

It would be actually quick to implement. That is so because Nodezator has a lot of tools to deal with this kind of geometry manipulation. I didn't do it yet due to more urgent features, but it will eventually be added. I intend to publish a roadmap in the near feature so people can visualize more easily what is planned for Nodezator's future.
